src/pretix/plugins/stripe/payment.py

```diff
@@ -865,6 +872,213 @@ def _confirm_payment_intent(self, request, payment):
                                      'with us if this problem persists.'))
 
 
+class StripeSEPADirectDebit(StripeMethod):
+    identifier = 'stripe_sepa_debit'
+    verbose_name = _('SEPA Debit via Stripe')
+    public_name = _('SEPA Debit')
+    method = 'sepa_debit'
+
+    def payment_form_render(self, request) -> str:
+        template = get_template('pretixplugins/stripe/checkout_payment_form_sepadirectdebit.html')
+        ctx = {
+            'request': request,
+            'event': self.event,
+            'settings': self.settings,
+            'form': self.payment_form(request),
+        }
+        return template.render(ctx)
+
+    @property
+    def payment_form_fields(self):
+        return OrderedDict([
+            ('iban', forms.CharField(label=_('IBAN'))),
+            ('accountname', forms.CharField(label=_('Account Holder Name'))),
+            ('accountemail', forms.CharField(label=_('Account Holder Email'))),
```

Comment:
The documentation says we need an email address, normally the account holder email address, this may vary from the order email or from the participant email, but we could prefill this maybe?

Reply:
Yeah, I think payment.order.emails hould be good enough. Email addresses are not really something banks check against for fraud prevention. (My bank doesn't even have my email address.)

Reply:
It is a Stripe requirement, I wouldn't expect them to verify that with the bank details, however from the wording and definition you may think that.
The reason why I would keep this different, as for companies, the bookkeeping department or financial department would be handling this, and I think if there is any doubt or problems, this email gets triggered and may contain sensitive data.
So I will prefill this with 'payment.order.email', however I would still give people the option to change it